tests/lib/Controller/SettingsController.php

```diff
@@ -486,4 +486,28 @@ public function passwordConfirmationAnnotation(): DataResponse {
 	public function passwordConfirmationAttribute(): DataResponse {
 		return new DataResponse();
 	}
+
+	/**
+	 * Route with oneOf
+	 *
+	 * @return DataResponse<Http::STATUS_OK, string|int|float|double|bool, array{}>
+	 *
+	 * 200: OK
+	 */
+	#[PasswordConfirmationRequired]
+	public function oneOf(): DataResponse {
+		return new DataResponse();
+	}
+
+	/**
+	 * Route with oneOf
+	 *
+	 * @return DataResponse<Http::STATUS_OK, array{test: string}|array{test: string, abc: int}, array{}>
```

Comment:
Can you also add a sample for:
```
	 * @return DataResponse<Http::STATUS_OK, array{foobar: string}|array{test: string, abc: int}, array{}>
```

Reply:
My current detection will still use anyOf for this case, even though these schemas would be fine.

Reply:
yeah, just want to have it "protocoled" so that future changes can be seen in effect